src/containers/my-quizzes/QuizzesContainer.tsx

```diff
@@ -27,6 +29,7 @@ const TestsContainer = () => {
   const { openDialog } = useConfirm()
   const searchTitle = useRef<string>('')
   const breakpoints = useBreakpoints()
+  const { page, handleChangePage } = usePagination()
   const itemsPerPage = getScreenBasedLimit(breakpoints, itemsLoadLimit)
   const columnsToShow = ajustColumns(breakpoints, columns, removeColumnRules)
```

Comment:
I guess you should add skip: (page - 1) * itemsPerPage to getQuizzes use callback below